Component/Product/Validator.php

```diff
@@ -125,7 +125,7 @@ public function getImportRowFailures(Importer $import, $importLines)
     {
         $failedRows = [];
         // Creates a validation model and runs the import data through so we can find which rows would fail
-        $validation = $import->createImportModel();
+        $validation = $import->getImportModel();
```

Comment:
this change implies we won't be backward compatible @paulpartington. Was not sure where the change came from to be honest.